src/components/modals/MetaMaskNotificationModal.vue

```diff
@@ -204,8 +257,19 @@ export default {
 
           break
 
-        // transaction submitted, waiting for mine
+        // @todo: validate this with savvy users
```

Comment:
`@TODO` comments should always be capitalized (my editor doesn't highlight them otherwise)